mne/inverse_sparse/tests/test_gamma_map.py

```diff
@@ -87,4 +88,38 @@ def test_gamma_map():
     _check_stc(stc, evoked, 85739, 20)
 
 
+@pytest.mark.slowtest
+@testing.requires_testing_data
+def test_gamma_map_vol_sphere():
+    """Gamma MAP with a sphere forward and volumic source space"""
+    evoked = read_evokeds(fname_evoked, condition=0, baseline=(None, 0),
+                          proj=False)
+    evoked.resample(50, npad=100)
+    evoked.crop(tmin=0.1, tmax=0.16)  # crop to nice window near samp border
```

Comment:
samp border? I'm not sure what this means... it's not near the border of the resampling operation, which would be at -0.2 and 0.5 (or maybe 1.0)?